packages/plugins/validation-cache/src/index.ts

```diff
@@ -2,9 +2,16 @@ import { Plugin } from '@envelop/types';
 import { GraphQLError, print } from 'graphql';
 import lru from 'tiny-lru';
 
+interface Cache<T> {
+  get(key: string): T | undefined;
+  set(key: string, value: T): void;
+  clear(): void;
+}
+
 export type ValidationCacheOptions = {
   max?: number;
   ttl?: number;
+  cache?: Cache<readonly GraphQLError[]>;
 };
```

Comment:
Maybe we should export this as a ValidationCache and hard-code the `readonly GraphQLError[]` type?

Reply:
👍  473d892